PendingReleaseNotes.md

```diff
@@ -11,6 +11,7 @@ v1.5...
 ### Ceph
 
 - Ceph mons require an odd number for a healthy quorum. An even number of mons is now disallowed.
+- preservePoolOnDelete is deprecated and is no longer allowed because of data-loss concerns. preserveFilesystemOnDelete acts as a replacement and preserves the whole filesystem when CephFilesystem CRD is deleted (which implicitly includes all associated pools). See [#6495](https://github.com/rook/rook/pull/6495) for more details.
```

Comment:
`preservePoolOnDelete` is still used for the CephBlockPool and CephObjectStore CRs, so let's clarify this statement is only true for the filesystem.

Reply:
Done